include/range/v3/view/exclusive_scan.hpp

```diff
@@ -0,0 +1,164 @@
+/// \file
+// Range v3 library
+//
+//  Copyright Mitsutaka Takeda 2018-present
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+//
+
+#ifndef RANGE_V3_VIEW_EXCLUSIVE_SCAN_HPP
+#define RANGE_V3_VIEW_EXCLUSIVE_SCAN_HPP
+
+
+#include <range/v3/utility/concepts.hpp>
+#include <range/v3/view_adaptor.hpp>
+#include <range/v3/view/view.hpp>
+
+namespace ranges {
+    inline namespace v3 {
+        /// \addtogroup group-views
+        /// @{
+        template<typename Rng, typename T, typename Fun>
+        struct exclusive_scan_view
+                : view_adaptor<exclusive_scan_view<Rng, T, Fun>, Rng> {
+        private:
+            friend range_access;
+            T init_;
+            semiregular_t<Fun> fun_;
+            using single_pass = SinglePass<iterator_t<Rng>>;
```

Comment:
This view cannot be multipass without also requiring that the type of the "accumulator" `sum_` in the adaptor is constructible from `const T&` as necessary to call `begin` repeatedly.